Makefile

```
@@ -74,15 +77,64 @@ build: clean
 	@echo "Building library..."
 	npm run build
 
+# Update version in package.json
+.PHONY: version
+version:
```

checkmake (0.2.2)

[warning] Target body for "version" exceeds allowed length of 5 (7).

(maxbodylength)